web-client/src/modules/requests/constants.tsx

```diff
@@ -7,11 +7,16 @@ import * as Yup from 'yup';
 export const PostListLocation = new Location('/requests/list/:type', {
   type: Yup.string(),
 });
+// ?? todo remove Accepted and Open
+export const offersPostType = 'offers';
+export const offersLocationStr = '/requests/'+offersPostType;
+export const requestsPostType = 'requests';
+export const requestsLocationStr = '/requests/'+requestsPostType;
+
 export const AcceptedRequestsLocation = new Location('/requests/accepted');
 export const FindRequestsLocation = new Location('/requests/find');
-export const OpenRequestsLocation = new Location('/requests/open');
+export const OpenRequestsLocation = new Location('/requests/opxxxen');
```

Comment:
There's a typo here and also this route is no longer useful to us. Could you remove it?